src/nccl_ofi_scheduler.c

```diff
@@ -42,6 +42,16 @@ static inline int set_schedule_by_threshold(nccl_net_ofi_threshold_scheduler_t *
 	/* Number of stripes is atleast 1 for zero-sized messages and at most equal to num of rails */
 	int num_stripes =
 		(int)NCCL_OFI_MAX(1, NCCL_OFI_MIN(NCCL_OFI_DIV_CEIL(size, scheduler->min_stripe_size), (unsigned)num_rails));
```

Comment:
Not related to this commit but spotted during review: It seems to me that `num_rails` should be `unsigned int` to begin with rather than signed.

Reply:
Correct. There was some previous discussion in the original PR to make num_rails as unsigned int, but this will involve changing it everywhere in the code base.
I was thinking to make a separate PR for this change. But please advise if required, I can add it as an additional commit to this PR.

Reply:
I didn't meant to do this change as part of this PR. I implied that we should change that in a separate PR.